apps/home/templatetags/custom_metadata_tags.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from wagtail.core.models import Site
+from wagtailmetadata import tags
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def custom_meta_tags(context, model=None):
+    """Use metadata from homepage if not set on current page.
+
+    Like https://github.com/neon-jungle/wagtail-metadata/blob/master/
+    wagtailmetadata/templatetags/wagtailmetadata_tags.py
+    but uses homepage meta tags if current site does not have own tags.
+    """
+    request = context.get('request', None)
+    if not model:
+        model = context.get('self', None)
+        if not hasattr(model, 'get_meta_title'):
```

Comment:
we should also ask if `get_meta_description` exists and is filled in. But have to fix the migrations first before I can try.